src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -40,16 +49,44 @@ public RobotContainer() {
   }
 
   private void setDefaultCommands() {
+
     // drivetrain.setDefaultCommand(new TeleopDrive(
     //   drivetrain,
     //   () -> ProcessedAxisValue(driverController, Axis.kLeftY)),
     //   () -> ProcessedAxisValue(driverController, Axis.kLeftX)),
     //   () -> ProcessedAxisValue(driverController, Axis.kRightX)),
     //   () -> driverController.getRawButton(OI.Driver.slowDriveButton)
     // ));
+
+    arm.setDefaultCommand(new ArmTeleop(arm, () -> inputProcessing(getStickValue(manipulatorController, Axis.kLeftY))));
+  }
+  private void setBindingsDriver() {
+    new JoystickButton(driverController, Button.kX.value).whileTrue(arm.sysIdQuasistatic(SysIdRoutine.Direction.kForward));
+    new JoystickButton(driverController, Button.kY.value).whileTrue(arm.sysIdQuasistatic(SysIdRoutine.Direction.kReverse));
+    new JoystickButton(driverController, Button.kB.value).whileTrue(arm.sysIdDynamic(SysIdRoutine.Direction.kForward));
+    new JoystickButton(driverController, Button.kA.value).whileTrue(arm.sysIdDynamic(SysIdRoutine.Direction.kReverse));
+    // 4 cardinal directions on arrowpad
+		// slowmode toggle on trigger
+		// 3 cardinal directions on letterpad
+  }
+  private void setBindingsManipulator() {
+		// intake/outtake on triggers
+    //Left trigger will be intake whilst right trigger will be outtake
+    //3 setpositions of arm on letterpad
+    //Up is Speaker, down is ground, right is Amp
+    //right joystick used for manual arm control
+
+    // Speaker Buttons
+    new JoystickButton(manipulatorController, Constants.OI.Manipulator.RAISE_TO_SPEAKER_POD_BUTTON).onTrue(new InstantCommand(() -> {arm.setArmTarget(PODIUM_ANGLE);}));
+    new JoystickButton(manipulatorController, Constants.OI.Manipulator.RAISE_TO_SPEAKER_NEXT_BUTTON).onTrue(new InstantCommand(() -> {arm.setArmTarget(SUBWOFFER_ANGLE);}));
+    new JoystickButton(manipulatorController, Constants.OI.Manipulator.RAISE_TO_SPEAKER_SAFE_BUTTON).onTrue(new InstantCommand(() -> {arm.setArmTarget(SAFE_ZONE_ANGLE);}));
+    // Amp and Intake Buttons
+    new JoystickButton(manipulatorController, Constants.OI.Manipulator.RAISE_TO_AMP_BUTTON).onTrue(new InstantCommand(() -> {arm.setArmTarget(AMP_ANGLE);}));
+    new JoystickButton(manipulatorController, Constants.OI.Manipulator.RAISE_TO_GROUND_BUTTON).onTrue(new InstantCommand(() -> {arm.setArmTarget(INTAKE_ANGLE);}));
+    // Cimber Buttons
+    new JoystickButton(manipulatorController, Constants.OI.Manipulator.RAISE_TO_CLIMBER_BUTTON).onTrue(new InstantCommand(() -> {arm.setArmTarget(CLIMBER_UP_ANGLE);}));
+    new JoystickButton(manipulatorController, Constants.OI.Manipulator.LOWER_TO_CLIMBER_BUTTON).onTrue(new InstantCommand(() -> {arm.setArmTarget(CLIMBER_DOWN_ANGLE);}));
```

Comment:
Make sure that the sysid buttons don't share the same buttons as these. You could also just comment them out when doing sysid testing and comment out sysid when using the arm.